src/hash/blake/tests.rs

```diff
@@ -18,3 +34,14 @@ proptest! {
         Blake3_256::hash(vec);
     }
 }
+
+// HELPER FUNCTIONS
+// ================================================================================================
+
+fn compute_expected_element_hash(elements: &[Felt]) -> blake3::Hash {
+    let mut bytes = Vec::new();
+    for element in elements.iter() {
+        bytes.extend_from_slice(&element.as_int().to_le_bytes());
+    }
+    blake3::hash(&bytes)
+}
```

Comment:
nit: this can be simplified to 

```rust
fn compute_expected_element_hash(elements: &[Felt]) -> blake3::Hash {
    blake3::hash(Felt::elements_as_bytes(elements))
}
```

Or even removed, as a one-liner will do the trick

Reply:
update: forget, `elements_as_bytes` will not provide canonical representations